src/api/Folder.js

```diff
@@ -217,13 +213,14 @@ class Folder extends Item {
         this.errorCode = ERROR_CODE_FETCH_FOLDER;
         let params = { fields: requestFields.toString() };
         if (!noPagination) {
-            params = Object.assign({}, params, {
+            params = {
```

Comment:
LOL, this seems harder to understand than before. Not to start an argument over the eslint rules, but... 🤷‍♂

Reply:
It's the same logic.